src/CoreShop2VueStorefrontBundle/Resources/config/services.yml

```diff
@@ -15,24 +15,17 @@ services:
         public: true
         tags: ['controller.service_arguments']
 
-    CoreShop2VueStorefrontBundle\Bridge\EnginePersisterFactory: ~
-
-    CoreShop2VueStorefrontBundle\Bridge\EnginePersister:
-        autoconfigure: false
-        arguments:
-            - '@es.manager.default'
-
     CoreShop2VueStorefrontBundle\Bridge\DocumentMapper\DocumentProductMapper: ~
 
     CoreShop2VueStorefrontBundle\Bridge\DocumentMapper\DocumentCategoryMapper: ~
 
-    CoreShop2VueStorefrontBundle\Bridge\DocumentMapper\DocumentAttributeMapper: ~
+    # CoreShop2VueStorefrontBundle\Bridge\DocumentMapper\DocumentAttributeMapper: ~
```

Comment:
Still need to figure out how these are supposed to work. We don't use them, maybe we can merge with them commented out for now and somebody else contributes that part.